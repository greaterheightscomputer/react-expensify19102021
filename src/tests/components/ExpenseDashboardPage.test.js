import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboard";

test("should render ExpenseDashboard correctly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
