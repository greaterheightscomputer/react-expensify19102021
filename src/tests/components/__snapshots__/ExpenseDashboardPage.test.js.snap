// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboard correctly 1`] = `
<div>
  This is from my dashboard component.
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
